### PR TITLE
MVP-238 Refining Techdocs

### PR DESCRIPTION
**Pull request by TaylorFries**
## 🎯 Summary

  
[MVP-238](https://citz-imb.atlassian.net/browse/MVP-238)

Updating formatting and links of techdocs. 

To see changes in place please reference: https://dev.developer.gov.bc.ca/docs/default/component/citz-imb-sso-express-npm-package/

## 🔰 Checklist

- [x] I have read and agree with the following checklist and am following the guidelines in our [Code of Conduct](/CODE_OF_CONDUCT.md) document.

> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - I have consulted with the team if introducing a new dependency.
> - My changes generate no new warnings.

### CODE REVIEW

File: techdocs/docs/using-the-package/environment-variables.md [hunk 1]
```diff
@@ -20,3 +20,14 @@ COOKIE_DOMAIN= # (optional) The domain for the refresh_token cookie.
 SM_LOGOUT_URL= # (optional) Site minder logout url.
 # By default, set to 'https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi'.
 ```
+
+!!! tip "Tip"
+    `SSO_CLIENT_ID`, `SSO_CLIENT_SECRET`, and `SSO_AUTH_SERVER_URL` details can be found in the `Technical Details` tab on your SSO integration dashboard (managed by Common Hosted Single Sign-On ([CSS]) team). 
+    resource -> SSO_CLIENT_ID,
```
Comment (BradyMitch) [inline techdocs/docs/using-the-package/environment-variables.md:26]: These dont show up on their own lines. Maybe we could try using <br />

**Review by BradyMitch — COMMENTED** [review]
We should put "Environment Variables" under "Getting Started" as its part of the set up process.

File: techdocs/docs/using-the-package/environment-variables.md (hunk 1 above)
Comment (TaylorFries) [reply techdocs/docs/using-the-package/environment-variables.md:27]: trying this on newest commit

File: techdocs/docs/getting-started/environment-variables.md [hunk 2]
```diff
@@ -20,3 +20,14 @@ COOKIE_DOMAIN= # (optional) The domain for the refresh_token cookie.
 SM_LOGOUT_URL= # (optional) Site minder logout url.
 # By default, set to 'https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi'.
 ```
+
+!!! tip "Tip"
+    `SSO_CLIENT_ID`, `SSO_CLIENT_SECRET`, and `SSO_AUTH_SERVER_URL` details can be found in the `Technical Details` tab on your SSO integration dashboard (managed by Common Hosted Single Sign-On ([CSS]) team). 
+    - resource -> SSO_CLIENT_ID,
+    - credentials.secret -> SSO_CLIENT_SECRET,
+    - auth-server-url -> SSO_AUTH_SERVER_URL
```
Comment (BradyMitch) [inline techdocs/docs/getting-started/environment-variables.md:28]:
```suggestion
    - resource -> SSO_CLIENT_ID,  
    - credentials.secret -> SSO_CLIENT_SECRET,  
    - auth-server-url -> SSO_AUTH_SERVER_URL  
```

File: techdocs/docs/getting-started/environment-variables.md [hunk 3]
```diff
@@ -20,3 +20,14 @@ COOKIE_DOMAIN= # (optional) The domain for the refresh_token cookie.
 SM_LOGOUT_URL= # (optional) Site minder logout url.
 # By default, set to 'https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi'.
 ```
+
+!!! tip "Tip"
+    `SSO_CLIENT_ID`, `SSO_CLIENT_SECRET`, and `SSO_AUTH_SERVER_URL` details can be found in the `Technical Details` tab on your SSO integration dashboard (managed by Common Hosted Single Sign-On ([CSS]) team).
```
Comment (BradyMitch) [inline techdocs/docs/getting-started/environment-variables.md:25]:
```suggestion
    `SSO_CLIENT_ID`, `SSO_CLIENT_SECRET`, and `SSO_AUTH_SERVER_URL` details can be found in the `Technical Details` tab on your SSO integration dashboard (managed by Common Hosted Single Sign-On ([CSS]) team).   
```

**Review by BradyMitch — APPROVED** [review]

**Pull request merged**